tasks/deploy.yml

```diff
@@ -10,7 +10,7 @@
 
     - name: Update the Preferences Server config file's CouchDB address
       replace:
-        dest: /tmp/gpii.preferencesServer.config.production.json
+        dest: '/tmp/{{ preferences_server_environment }}.json'
```

Comment:
It looks like the ``NODE_ENV`` environment variable needs to be used to point to this new file but the real issue is most likely this old default:

https://github.com/gpii-ops/docker-preferences-server/blob/master/provisioning/run-vars.yml#L5

[Line 9](https://github.com/gpii-ops/docker-preferences-server/blob/master/provisioning/run-vars.yml#L9) might also need to be updated depending on what is the preferred way of starting the Preferences Server.